vms/volunteer/models.py

```diff
@@ -103,7 +103,7 @@ class Volunteer(models.Model):
         blank=True,
         validators=[
             RegexValidator(
-                r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\.)|(,)|(\-)|(!)]+$',
+               r'^(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9]\.[^\s]{2,})+$',

```

Comment:
I'm don't think a url regex would work great for resume field, what we can do it to allow user only to upload some specific file type for resume. Can you please share more on how you think this would work for resume? also there are few others regexes which can be improved in this file.

Reply:
@tapasweni-pathak IMO it would be better if we use file field inside the model rather than text field for resume. Also we can then use validators to check that files with certain allowed extensions are uploaded. Check this for reference [link](https://stackoverflow.com/questions/3648421/only-accept-a-certain-file-type-in-filefield-server-side)

Reply:
@tapasweni-pathak Since this issue asked to modify the resume field (text type), I created a separate issue to edit upload_resume field to add above functionality. Please check [#565](https://github.com/systers/vms/issues/565).

Reply:
I am sorry I had to close the issue because upload_resume field works file for uploading files. So this field i.e. resume (text type) would be better with a URL validator only. What do you think @tapasweni-pathak

Reply:
heyo, taking this to Slack.

Reply:
Okay :)